src/gui/qgsexpressionbuilderwidget.cpp

```diff
@@ -1597,22 +1600,35 @@ void QgsExpressionBuilderWidget::loadExpressionsFromJson( const QJsonDocument &e
     }
   }
 
-  QgsLogger::warning( "" + QStringLiteral( __FILE__ ) + ": " + QString::number( __LINE__ ) );
   loadUserExpressions( );
+
+  QgsLogger::warning( QString::number( skippedExpressionLabels.count() ) + QStringLiteral( __FILE__ ) + ": " + QString::number( __LINE__ ) );
+
+  if ( ! skippedExpressionLabels.isEmpty() )
+  {
+    QStringList skippedExpressionLabelsQuoted;
+    for ( const QString &skippedExpressionLabel : skippedExpressionLabels )
+      skippedExpressionLabelsQuoted.append( QString( "'%1'" ).arg( skippedExpressionLabel ) );
+
+    QMessageBox::information( this,
+                              tr( "Skipped expressions" ),
```

Comment:
Maybe "Invalid Expressions" would be better? (Also not title case capitalisation for dialog titles)

Reply:
They could be perfectly valid expressions, just skipped to import because of label collision. More like "Not Imported Expressions"? "Skipped Imports"?